### PR TITLE
global: dispose database connection when closing

### PR DESCRIPTION
**Pull request by egabancho**
Signed-off-by: Esteban J. G. Gabancho <esteban.gabancho@gmail.com>

### CODE REVIEW

File: invenio_celery/__init__.py [hunk 1]
```diff
@@ -130,7 +130,10 @@ def read_configuration(self):
 
     def close_database(self, **dummy_kwargs):
         if self.db:
+            from flask import has_app_context
```
Comment (jirikuncar) [inline invenio_celery/__init__.py]: this can be a global import

File: invenio_celery/__init__.py [hunk 2]
```diff
@@ -130,7 +130,10 @@ def read_configuration(self):
...
             self.db.session.remove()
+            if has_app_context:
```
Comment (jirikuncar) [inline invenio_celery/__init__.py]: `if has_app_context():`

Comment (jirikuncar) [inline invenio_celery/__init__.py]: http://flask.pocoo.org/docs/0.10/api/#flask.has_app_context

**Comment by egabancho** [comment]
ping @jirikuncar

**Comment by jirikuncar** [comment]
@egabancho `global: database connection disposal on closing` + `* BETTER Closes database connection ...`

**Comment by egabancho** [comment]
ping @jirikuncar

**Pull request merged**